lib/blueprinter-activerecord/preloader.rb

```diff
@@ -69,12 +73,25 @@ def pre_render(object, blueprint, view, options)
     #
     def self.preloads(blueprint, view_name, model=nil)
       view = blueprint.reflections.fetch(view_name)
-      view.associations.each_with_object({}) { |(_name, assoc), acc|
+      preload_vals = view.associations.each_with_object({}) { |(_name, assoc), acc|
+        # look for a matching association on the model
         ref = model ? model.reflections[assoc.name.to_s] : nil
         if (ref || model.nil?) && !assoc.blueprint.is_a?(Proc)
           ref_model = ref && !(ref.belongs_to? && ref.polymorphic?) ? ref.klass : nil
           acc[assoc.name] = preloads(assoc.blueprint, assoc.view, ref_model)
         end
+
+        # look for a :preload option on the association
```

Comment:
Is this necessary?  I'm struggling to think of a situation for this.  If so, we should add that to the README too

Reply:
I've seen associations that use blocks that call arbitrary code. Rare, but they exist.